applications/tasks.py

```diff
@@ -0,0 +1,136 @@
+            application = application_section.application
+            application_round = application.application_round
+
+            common_details = ReservationDetails(
+                name=recurring_reservation.name,
+                description=application.additional_information,
+                type=ReservationTypeChoice.SEASONAL,
+                user=recurring_reservation.user,
+                handled_at=application_round.handled_date,
+                num_persons=application_section.num_persons,
+                buffer_time_before=datetime.timedelta(0),
+                buffer_time_after=datetime.timedelta(0),
+                reservee_first_name=application.contact_person.first_name,
+                reservee_last_name=application.contact_person.last_name,
+                reservee_email=application.contact_person.email,
+                reservee_phone=application.contact_person.phone_number,
+                billing_address_street=application.billing_address.street_address,
+                billing_address_city=application.billing_address.city,
+                billing_address_zip=application.billing_address.post_code,
+            )
+
+            common_details["reservee_type"] = (
+                CustomerTypeChoice.INDIVIDUAL
+                if application.applicant_type == ApplicantTypeChoice.INDIVIDUAL
+                else CustomerTypeChoice.BUSINESS
+                if application.applicant_type == ApplicantTypeChoice.COMPANY
+                else CustomerTypeChoice.NONPROFIT
+            )
+
+            if common_details["reservee_type"] == CustomerTypeChoice.INDIVIDUAL:
+                common_details["reservee_address_street"] = application.billing_address.street_address
+                common_details["reservee_address_city"] = application.billing_address.city
+                common_details["reservee_address_zip"] = application.billing_address.post_code
+
+            else:
+                common_details["reservee_organisation_name"] = application.organisation.name
+                common_details["reservee_id"] = application.organisation.identifier
+                common_details["reservee_is_unregistered_association"] = application.organisation.identifier is None
+                common_details["reservee_address_street"] = application.organisation.address.street_address
+                common_details["reservee_address_city"] = application.organisation.address.city
+                common_details["reservee_address_zip"] = application.organisation.address.post_code
+
+            recurring_reservation.actions.bulk_create_reservation_for_periods(
+                periods=slots.possible,
+                reservation_details=(
+                    common_details
+                    | ReservationDetails(
+                        state=ReservationStateChoice.CONFIRMED,
+                    )
+                ),
+            )
+            recurring_reservation.actions.bulk_create_reservation_for_periods(
```

Comment:
This part can be removed (reservation not created for the impossible slots), otherwise this PR is good to go.